targets/kodi

```diff
@@ -5,27 +5,19 @@
 if [ "${TARGETNOINSTALL:-c}" = 'c' ]; then
     if [ "$DISTRO" = 'kali' ]; then
         error 99 "Kodi not supported on Kali Linux."
+    elif [ "$DISTRO" = 'debian' ]; then
```

Comment:
Seems to be current in [sid](https://packages.debian.org/sid/video/kodi) (17), and jessie-backports seems to be version 16, which doesn't seem that old.  Can you get it to work so we don't artificially limit targets?

Generally we avoid creating targets that are only available in certain distros (unity being the obvious exception to that rule).